### PR TITLE
Fix port numbers.

### PR DESCRIPTION
**Pull request by amackworth**
Fixing port numbers to get the code to actually run on the robot!

### CODE REVIEW

**Comment by amackworth** [comment]
@col6y Is there any way of accessing the MXP expansion ports from the CCRE? (The Clamp encoder is connected to MXP inputs 1 and 2, and, for right now, I'm just setting it to always 0.)

File: QuasarHelios2015/src/org/team1540/quasarhelios/AutoLoader.java [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 public class AutoLoader extends InstinctModule {
     private final BooleanStatus running;
-    public static final BooleanInput crateInPosition = BooleanMixing.createDispatch(Igneous.makeDigitalInput(3), Igneous.globalPeriodic);
+    public static final BooleanInput crateInPosition = BooleanMixing.createDispatch(Igneous.makeDigitalInput(5), Igneous.globalPeriodic);
```
Comment (celskeggs) [inline QuasarHelios2015/src/org/team1540/quasarhelios/AutoLoader.java:13]: As of last time I was in, Anna told me that they'd put the limit switches in 2 and 4, and left the crate port as 3.

Assuming that this is spurred by getting Helios to work, why did they change it?

File: QuasarHelios2015/src/org/team1540/quasarhelios/Clamp.java [hunk 2]
```diff
@@ -23,11 +23,12 @@
 
     public static void setup() {
 
-        FloatInputPoll encoder = Igneous.makeEncoder(4, 5, false);
+//        FloatInputPoll encoder = Igneous.makeEncoder(1, 2, false);
+        FloatInputPoll encoder = new FloatStatus(0.0f);
```
Comment (celskeggs) [inline QuasarHelios2015/src/org/team1540/quasarhelios/Clamp.java]: You want `FloatMixing.always`.

**Comment by celskeggs** [comment]
The MXP DIO ports start at "port" 10.

**Comment by celskeggs** [comment]
To my knowledge.

**Comment by amackworth** [comment]
Mm, ok. Thanks!

File: QuasarHelios2015/src/org/team1540/quasarhelios/AutoLoader.java (hunk 1 above)
Comment (amackworth) [inline QuasarHelios2015/src/org/team1540/quasarhelios/AutoLoader.java:13]: No idea.

**Comment by celskeggs** [comment]
FYI I'll be later than expected thanks to a bus system reroute.

File: QuasarHelios2015/src/org/team1540/quasarhelios/Rollers.java [hunk 3]
```diff
@@ -37,7 +37,11 @@
 
     public static void setup() {
         FloatMixing.pumpWhen(QuasarHelios.globalControl, FloatMixing.addition.of((FloatInputPoll) spinOverride, motorSpeed), allRollers);
-
+        Igneous.joystick3.getButtonSource(1).send(new EventOutput() {
+            public void event() {
+                frontRollers.set(1.0f);
+            }
+        });
```
Comment (celskeggs) [inline QuasarHelios2015/src/org/team1540/quasarhelios/Rollers.java]: Why do we have this?

**Comment by celskeggs** [comment]
Looks good.

**Pull request merged**